<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>340</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>업무</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>340</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CMS
Fool proof</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>664</x>
      <y>340</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bluecon
프로그램</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>270</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>출두일자별
도면 폴더</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>270</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>전기도면
회사 서버</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>664</x>
      <y>270</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>전기도면
연구소 서버</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>200</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DT기준</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>200</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>모니터
AS 폴더</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>664</x>
      <y>200</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>블루콘
통합옵션도면</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>410</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>통합용량표(서버)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>664</x>
      <y>410</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>모니터
Screen shot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Q">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>200</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>전기도면
폴더</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_W">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>200</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>용량표
폴더</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_E">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>200</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>번역
엑셀파일</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_R">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>200</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>모터정격
전류</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>180</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>180</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>180</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>308</x>
      <y>180</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number Pad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_A">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>290</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bluecon1
프로젝트</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_S">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>290</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bluecon2
프로젝트</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_D">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>290</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trunk</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>270</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>270</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>270</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_F">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>290</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>메뉴얼</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Z">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>380</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>출하사진</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_X">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>380</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sequence
Chart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_C">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>380</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KCS
인증폴더</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>360</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_F3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>히터 규격</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>F3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_F2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>휴게소</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>F2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_F1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Share
Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>F1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_F4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>언어번역
프로그램</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>F4</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
